### PR TITLE
Create footer

### PR DESCRIPTION
**Pull request by KirilushkinAleksandr**

### CODE REVIEW

File: client/src/components/footer/links.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import './footer.scss';
+
+class Links extends React.Component {
```
Comment (miffn3) [inline client/src/components/footer/links.js]: maybe footer links? Or change style sheets

File: client/src/App.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import './App.scss';
+import Footer from './components/footer';
 
 function App() {
   return (
     <div className="App">
+      <Footer />
```
Comment (fastndead) [inline client/src/App.js]: we do not change App component yet

Comment (KirilushkinAleksandr) [reply client/src/App.js]: Ok, that won't be pushed

File: client/src/components/footer/links.js (hunk 1 above)
Comment (KirilushkinAleksandr) [reply client/src/components/footer/links.js]: Links are incapsulated in footer, we discussed it

**Pull request merged**